checker/check.py

```diff
@@ -0,0 +1,254 @@
+from django.conf import settings
+from django.db import models
+
+from github import Github
+from cloudfoundry_client.client import CloudFoundryClient
+from datetime import datetime
+import pytz
+import yaml
+import csv
+from .models import PipelineApp, PipelineEnv
+
+import logging
+
+log = logging.getLogger(__name__)
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+
+def get_pipeline_configs(repo):
+    yaml_file_list=[]
+    for content_file in repo.get_contents(""):
+        if ".yaml" in content_file.path:
+            yaml_file_list.append(content_file.path)
+    return yaml_file_list  
+
+
+def get_app_config_yaml(repo, config_file):
+    config_text = repo.get_contents(config_file).decoded_content.decode()
+    config_yaml = yaml.load(config_text, Loader=yaml.FullLoader)
+    for git_cleanup in settings.GIT_CLEANUP_LIST:
+        config_yaml["scm"] = config_yaml["scm"].replace(git_cleanup, "")
+    return config_yaml
+
+
+def get_config_environment_names(config_yaml):
+    environments=[]
+    for environment_yaml in config_yaml["environments"]:
+        environments.append(environment_yaml["environment"])
+    return environments
+
+
+def write_csv(write_mode, columns):
+    f = open(settings.CSV_OUTPUT_FILENAME, write_mode)
+    writer = csv.writer(f)
+    writer.writerow(columns)
+    f.close()
+
+def write_headers(App, Env):
```

Comment:
```suggestion
def write_headers(app, env):
```
the params should be lowercase otherwise it looks like you're dealing with classes

Reply:
Ah yes, that was intended. In that function I am using the the class definition (since when writing the headers - there are no instantiated objects yet). It get's called from [line 72](https://github.com/uktrade/cf-app-version-checker/blob/4e0cfd75b33580c85c59e9c08e9da71d980ffbe3/checker/check.py#L72):
```python
    write_headers(PipelineApp, PipelineEnv)
```

Reply:
yes I see that but I mean a reader might confuse App, and Env for real classes.

If you are always passing in `PipelineApp`, `PipelineEnv` then why do they need to be parameters at all?